test/parallel-benchmark/mzcompose.py

```diff
@@ -1133,56 +1143,72 @@ def less_than_is_regression(stat: str) -> bool:
 
 
 def check_regressions(
-    this_stats: dict[str, dict[str, Statistics]],
-    other_stats: dict[str, dict[str, Statistics]],
+    this_stats: dict[Scenario, dict[str, Statistics]],
+    other_stats: dict[Scenario, dict[str, Statistics]],
     other_tag: str,
 ) -> list[TestFailureDetails]:
     failures: list[TestFailureDetails] = []
 
     assert len(this_stats) == len(other_stats)
 
-    for scenario in this_stats.keys():
+    for scenario, other_scenario in zip(this_stats.keys(), other_stats.keys()):
+        scenario_name = type(scenario).__name__
+        assert type(other_scenario).__name__ == scenario_name
         has_failed = False
-        print(f"Comparing scenario {scenario}")
+        print(f"Comparing scenario {scenario_name}")
         output_lines = [
             f"{'QUERY':<40} | {'STAT':<7} | {'THIS':^12} | {'OTHER':^12} | {'CHANGE':^9} | {'THRESHOLD':^9} | {'REGRESSION?':^12}",
             "-" * 118,
         ]
 
+        ignore_regressions = set()
+        for phase in scenario.phases:
+            if not isinstance(phase, LoadPhase):
```

Comment:
Can you add an explaining comment here why this flag is ignored for `LoadPhase`?